tests/test_sunsch.py

```diff
@@ -725,40 +744,6 @@ def test_wrap_long_lines():
     )
 
 
-def test_long_vfp_lines(tmp_path, caplog, mocker):
-    """Make sure that lines can't get too long for Eclipse"""
-    os.chdir(tmp_path)
-    Path("vfp.inc").write_text(
-        """VFPPROD
-1 100 'GAS' 'WGR' 'GOR' 'THP' ' ' 'METRIC' 'BHP' /
-100000 300000 1000000 2000000 3000000 4000000 5000000
-6000000 7000000 8000000 9000000 10000000
-/
-30 50 /
-0 /
-5000 /
-0 /
--- This two-line record might get emitted as one long line by opm-common
--- This will error in Eclipse
-1 1 1 1 35.4324212 37.234245 39.2343242 40.4324234 43.23523546 45.54676535
-        47.3242356 49.345345 50.2342343 52.4353456 54.24342344 56.32424324
-/
-2 1 1 1 55.4324212 57.234245 59.2343242 60.4324234 63.23523546 65.54676535
-        67.3242356 69.345345 70.2342343 72.4353456 74.24342344 76.32424324
-/""",
-        encoding="utf-8",
-    )
-
-    Path("conf.yml").write_text(
-        "files:\n  - vfp.inc\noutput: sch.inc", encoding="utf-8"
-    )
-    mocker.patch("sys.argv", ["sunsch", "conf.yml"])
-    sunsch.main()
-    schinc = Path("sch.inc").read_text(encoding="utf8")
-    assert max([len(line) for line in schinc.split("\n")]) <= 129
-    assert "Line 7 had length 146, wrapped" in caplog.text
```

Comment:
This is the line that fails? I would nuke only this line and keep the rest.

Reply:
yes, that is the failing line.

Reply:
100% correct, re-introducing the test without the last line. Verified ok with opm 2021.10.